### ISSUE TITLE
How to get evaulation metrics from visualize.py

### ISSUE THREAD

**Issue by LTT-5** [thuml/Transfer-Learning-Library#273]
Hello, thanks again for your excellent work! 

I want to calculate evaluation metrics such as mAP and IoU for the test dataset (made by myself, similar to FoggyCityscapes) using **visualize.py**. How should I updated visualize.py code?

Hope to receive your feedback.

**Comment by LTT-5** [comment]
Hello, 

I also have a new problem. When training with d_adapt.py, **I encountered this error**: 
oserror: [WinError 1455] The page file is too small to complete the operation. Error loading “E:\miniconda3\envs\mmyolo\lib\site-packages\torch\lib\caffe2_detectron_ops_gpu.dll” or one of its dependencies.

**The code I am running is:**
python d_adapt.py --workers-c 1 --max-train-c 20 --ignored-scores-c 0.05 0.5 --confidence-ratio-c 0.1 --config-file d_adapt/config/faster_rcnn_R_101_C4_cityscapes.yaml -s Cityscapes datasets_/cityscapes_in_voc -t FoggyCityscapes datasets_/foggy_cityscapes_in_voc/ --test FoggyCityscapesTest datasets_/foggy_cityscapes_in_voc/ --finetune --trade-off 0.5 --bbox-refine OUTPUT_DIR logs/faster_rcnn_R_101_C4/cityscapes2foggy/phase2 MODEL.WEIGHTS logs/faster_rcnn_R_101_C4/cityscapes2foggy/phase1/model_final.pth SEED 0

Can you tell me how I should correct this, or turn down the batch_size somewhere?

Hope to receive your feedback.
